examples/demo.sh

```diff
@@ -30,6 +35,11 @@ docker-compose up -d superset
 echo "Sleeping for 30s"
 sleep 30
 
+if [ "$1" == "celery" ]; then
+  echo "Starting celery worker service..."
+  docker-compose up -d worker
+fi
```

Comment:
Is there a reason the worker comes up after superset?

Reply:
Thanks for mentioning this. It can actually start right after the superset. I can move this code after the command `docker-compose up -d superset`. Does it sound good to you?

Reply:
Perfect, thanks!

Reply:
Done. Thank you!